services/go_app_modules/authentication-api/ressources/jwt/service/create_jwt.go

```diff
@@ -0,0 +1,27 @@
+package service
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+)
+
+func (s *JwtService) CreateJWT(jwtData *JwtDataModel) (string, error) {
+	claims := jwt.MapClaims{
+		"user-uuid":    jwtData.UserUuid,
+		"session-uuid": jwtData.SessionID,
+		"exp":          time.Now().Add(time.Hour * 24).Unix(),
```

Comment:
Now its 10 Minutes valid, because Refresh Tokens are on their ways into this pr as well. How long would you suggest to make them valid?

Reply:
Could you name me a specific case, where this would be necessary? Like what would differ from a staging environment to production? Shouldn't it be consistent in terms of behaviour?